ragna/assistants/_api.py

```diff
@@ -11,12 +11,10 @@
 from ragna.core import Assistant, EnvVarRequirement, RagnaException, Requirement, Source
 
 
-class ApiAssistant(Assistant):
-    _API_KEY_ENV_VAR: str
-
+class _ApiAssistant(Assistant):
     @classmethod
     def requirements(cls) -> list[Requirement]:
-        return [EnvVarRequirement(cls._API_KEY_ENV_VAR), *cls._extra_requirements()]
+        return []
 
     @classmethod
     def _extra_requirements(cls) -> list[Requirement]:
```

Comment:
Not sure if it makes sense to have this on the superclass if for now only the authenticated subclass needs it.